lib/ui/shared/dialogs/remind_me_dialog/remind_me_dialog_viewmodel.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:awesome_notifications/awesome_notifications.dart';
+import 'package:hng/app/app.locator.dart';
+import 'package:hng/services/notification_service.dart';
+import 'package:intl/intl.dart';
+import 'package:stacked/stacked.dart';
+import 'package:stacked_services/stacked_services.dart';
+
+class RemindMeDialogViewModel extends BaseViewModel {
+  final navigationService = locator<NavigationService>();
+  final notificationService = locator<NotificationService>();
+  final now = DateTime.now();
+
+  String twentyMinutes = DateFormat('h:mm a')
+      .format(DateTime.now().add(const Duration(seconds: 10)));
+  var oneHour =
+      DateFormat('h:mm a').format(DateTime.now().add(const Duration(hours: 1)));
+  var threeHours =
+      DateFormat('h:mm a').format(DateTime.now().add(const Duration(hours: 3)));
+  var tomorrow = DateFormat("EEEE 'at' h:mm a")
+      .format(DateTime.now().add(const Duration(days: 1)));
+  var nextWeek = DateFormat("EEEE 'at' h:mm a")
+      .format(DateTime.now().add(const Duration(days: 7)));
+
+//twenty minutes
+  void messageRemindertwentyMinutes() async {
+    await AwesomeNotifications().createNotification(
```

Comment:
This should be in the notification service, as a function so that you only import the notification service and pass the required parameters when you want to create a new notification

Reply:
The same should be done everywhere in this file

Reply:
Done